packages/glimmer-runtime/lib/compiled/blocks.ts

```diff
@@ -29,12 +30,14 @@ export abstract class Block {
   public children: InlineBlock[];
   public program: Program;
   public symbolTable: SymbolTable;
+  public meta: StatementMeta = null;
   protected compiled: CompiledBlock = null;
 
   constructor(options: BlockOptions) {
     this.symbolTable = options.symbolTable || null;
     this.children = options.children;
     this.program = options.program;
+    this.meta = options.meta || {};
```

Comment:
Also, `BlockOptions.meta` is not optional, so all I had to do was just remove the ` || {}`